starter-code/index.html

```diff
@@ -2,29 +2,76 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css" />
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<div class = "banner">
+ <div> <!-- spotify image -->
+  <img class = "img-size col" src= "./images/spotify-logo.png"/>
+ </div>
+ 
+ <div>
+ <h3 class = "col-1">Premium</h3> 
```

Comment:
You should use a list for this kind of element, it will easier to style it : 

```html

<div id="header">
    <img src="images/spotify-logo.png" alt="spotify logo">
    <ul style>
      <li>
        <p>
          <a href="#">Premium</a>
        </p>
      </li>
      <li>
        <p>
          <a href="#">Discover</p>
        </p>
      </li>
      <li>
        <p>
          <a href="#">Help</p>
      </li>
      <li>
        <p>
          <a href="#">Download</a>
        </p>
      </li>
    </ul>
  </div>

```

Reply:
And for the css part : 

```css

#header {
  position: fixed;
  height: 90px;
  background-color: white;
  width: 100%;
  display: flex;
  box-shadow: 0px 10px 5px -8px rgba(0, 0, 0, 0.75);
}

#header img {
  height: 70%;
  margin: 10px 0 0 30px;
}

#header ul {
  display: flex;
  align-items: baseline;
  margin: 2% 0 0 50%;
}

#header li {
  margin: 0 20px;
  list-style-type: none;
}


```